### PR TITLE
Update react-bootstrap components

### PR DESCRIPTION
**Pull request by swashcap**
Asana task: https://app.asana.com/0/37131652076106/134376730776889

* **Problem:** react-bootstrap depreciated the `Input` component.
* **Solution:** Upgrade to `FormControl`, `FormGroup`, etc. _Bonus:_ update for new listing rules
* **Testing:** Pull changes and boot? Run through project forms? (see comments below)

### CODE REVIEW

**Comment by swashcap** [comment]
d05dc129be22fddf3dc15118ca685a361e9578b6 cleans up the components and updates the react-bootstrap form components. Unfortunately, I’m getting these errors when adding a new project:

whoa-errors

Any what’s up with this, @cdaringe?

**Comment by swashcap** [comment]
Also, there’s no analysis for my test consortium?

no-analysis-dog

@cdaringe, are you using some custom seed data for CouchDB?

**Comment by cdaringe** [comment]
@swashcap, the syncing was only _partially_ working with storage proxy.  Figured that out just today, haven't revved it

File: app/render/components/field-input.js [hunk 1]
```diff
@@ -1,104 +0,0 @@
...
-            if (error) {
-                return error;
-            }
-        };
-    }
-    render() {
-        return (
-            <Input ref="input" {...this.props} onChange={this.handleInputChange.bind(this)} />
-        );
-    }
```
Comment (cdaringe) [inline app/render/components/field-input.js]: Laaaateeeer old stuff

**Comment by cdaringe** [comment]
I think I know what's up with the errors.  The project SubAPI may not even exist in the latest client.  That workflow will be mo simple.  We will probably both arrive at patching it at the same time I bet.  Let's hash it out on slack

**Comment by swashcap** [comment]
:boom: 👍 Cool, sounds like a plan.

**Pull request merged**